### PR TITLE
Add release notes to README file.

### PR DESCRIPTION
**Pull request by keep94**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -306,3 +306,28 @@ sender.Close()
 
 * Reach out to us on our public [Slack channel](https://www.wavefront.com/join-public-slack).
 * If you run into any issues, let us know by creating a GitHub issue.
+
+## Releasing this SDK
```
Comment (oppegard) [inline README.md:310]: Please add this section to the Table of Contents at the top of the README.

**Review by oppegard — CHANGES_REQUESTED** [review]
Looks good. We should definitely automate most/all of this into a GitHub Action that we can manually trigger: https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow.

The input to the action would be the new version, e.g. `v0.10.3`.

File: README.md [hunk 2]
```diff
@@ -306,3 +306,28 @@ sender.Close()
...
+
+1. Make sure that all changes for the release are merged with the master branch
+2. Make sure that no applications are running that bind port 8080. See [Issue #102](https://github.com/wavefrontHQ/wavefront-sdk-go/issues/102)
```
Comment (shavidissa) [inline README.md:313]: Make sure that the application doesn't bind port 8080. See [Issue #102](https://github.com/wavefrontHQ/wavefront-sdk-go/issues/102).

File: README.md [hunk 3]
```diff
@@ -306,3 +306,28 @@ sender.Close()
...
+3. From the root directory of the project in the master branch, check that the following command run cleanly
+   1. `go install ./...`
+   2. `go test ./...`
+   3. `go vet ./...`
+4. Run `go mod tidy` to make sure it doesn't introduce new changes. If it brings new changes, check those in first before doing the release
+5. Find out the version of the last release of the SDK clicking on "Releases" in the about section on the right.
```
Comment (shavidissa) [inline README.md:319]: Click **Releases** in the about section to get details on the version history.

File: README.md [hunk 4]
```diff
@@ -306,3 +306,28 @@ sender.Close()
...
+6. Decide what the version of the next release will be. We follow [semantic versioning](https://semver.org/).
+7. Execute `git tag -a -m 'v0.10.3' 'v0.10.3'` replacing v0.10.3 with the new version.
+8. Push the tag up to github `git push upstream v0.10.3`
```
Comment (shavidissa) [inline README.md:322]: Push the tag up to GitHub `git push upstream v0.10.3`

File: README.md [hunk 5]
```diff
@@ -306,3 +306,28 @@ sender.Close()
...
+9. Log into github, click on "Releases" in the about section on the right, and click on "Draft a new release"
```
Comment (shavidissa) [inline README.md:323]: Log into GitHub, click **Releases** in the about section, and click **Draft a new release**.

Convention we follow: UI text needs to be in bold :)

File: README.md [hunk 6]
```diff
@@ -306,3 +306,28 @@ sender.Close()
...
+10. For version, choose the version you decided upon in step 6.
+11. Provide a short but descriptive title for the release
+12. Fill in the details of the release. Please copy the mark down from the previous release and follow the same format
```
Comment (shavidissa) [inline README.md:326]: Fill in the details of the release. Please copy the markdown from the previous release and follow the same format.

File: README.md [hunk 7]
```diff
@@ -306,3 +306,28 @@ sender.Close()
...
+   1. `go install ./...`
+   2. `go test ./...`
+   3. `go vet ./...`
+4. Run `go mod tidy` to make sure it doesn't introduce new changes. If it brings new changes, check those in first before doing the release
+5. Find out the version of the last release of the SDK clicking on "Releases" in the about section on the right.
+6. Decide what the version of the next release will be. We follow [semantic versioning](https://semver.org/).
+7. Execute `git tag -a -m 'v0.10.3' 'v0.10.3'` replacing v0.10.3 with the new version.
+8. Push the tag up to github `git push upstream v0.10.3`
+9. Log into github, click on "Releases" in the about section on the right, and click on "Draft a new release"
+10. For version, choose the version you decided upon in step 6.
```
Comment (shavidissa) [inline README.md:324]: For **version**, choose the version you decided upon in step 6.

File: README.md [hunk 8]
```diff
@@ -306,3 +306,28 @@ sender.Close()
...
+11. Provide a short but descriptive title for the release
+12. Fill in the details of the release. Please copy the mark down from the previous release and follow the same format
+13. Click on "Publish release"
```
Comment (shavidissa) [inline README.md:327]: Click **Publish release**.

File: README.md [hunk 9]
```diff
@@ -306,3 +306,28 @@ sender.Close()
...
+14. From your home directory run `mkdir -p go/src/cmd/scratch`
+15. Run `cd go/src/cmd/scratch`
+16. Run `go mod init`
+17. Run `go get github.com/wavefronthq/wavefront-sdk-go@v0.10.3` replacing v0.10.3 with the version you just published
+18. Step 17 lets pkg.go.dev know that there is a new version of the SDK to cache
```
Comment (shavidissa) [inline README.md:332]: Step 17 lets `pkg.go.dev` know that there is a new version of the SDK to cache.

File: README.md [hunk 10]
```diff
@@ -306,3 +306,28 @@ sender.Close()
...
+19. After 15 minutes or so check go to [pkg.go.dev](https://pkg.go.dev/github.com/wavefronthq/wavefront-sdk-go) to see if the version upgraded. Once the version upgrades on pkg.go.dev, other go users will get the new version you just published by default.
```
Comment (shavidissa) [inline README.md:333]: After 15 minutes, go to [pkg.go.dev](https://pkg.go.dev/github.com/wavefronthq/wavefront-sdk-go) and see if the version upgraded. Once the version upgrades on pkg.go.dev, all the go users get the new version you just published.

**Review by shavidissa — CHANGES_REQUESTED** [review]
Added some minor comments :) 
Minor nitpick: Can we add a full stop to all the sentences? Some have it and some don't.

File: README.md (hunk 1 above)
Comment (keep94) [reply README.md:310]: Done.

[the remark this answers is not in the document]
Comment (keep94) [reply README.md:310]: Done.

File: README.md (hunk 2 above)
Comment (keep94) [reply README.md:313]: I changed to "Make sure that no applications bind port 8080."

File: README.md (hunk 3 above)
Comment (keep94) [reply README.md:319]: Done.

File: README.md (hunk 4 above)
Comment (keep94) [reply README.md:322]: Done.

File: README.md (hunk 5 above)
Comment (keep94) [reply README.md:323]: Done.

File: README.md (hunk 6 above)
Comment (keep94) [reply README.md:326]: Done.

File: README.md (hunk 7 above)
Comment (keep94) [reply README.md:324]: Done.

File: README.md (hunk 10 above)
Comment (keep94) [reply README.md:333]: Done.

**Comment by keep94** [comment]
Thank you oppegard and shavidissa for your comments.

**Review by shavidissa — APPROVED** [review]
Lgtm

**Review by oppegard — APPROVED** [review]

**Pull request merged**